packages/db/src/core/remove.spec.ts

```diff
@@ -12,12 +12,18 @@ interface User {
 
 let db: Database;
 let userTable: Table<User, "id">;
+let userTableWithIndex: Table<User, "id">;
 
 beforeEach(async () => {
   db = createDB();
   userTable = createTable<User>(db, "users")();
+  userTableWithIndex = createTable<User>(
```

Comment:
If the table is just used in one test, you can create it in just that one test.
See "should update indexes" in `update.spec.ts`.